docs/included-checks.md

```diff
@@ -0,0 +1,58 @@
+# ESH Guidelines Covered
+
+## A. Code Style
+1. [The Java naming conventions should be used.](https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L80) - `severity=info`
+2. [Every Java file must have a license header. You can run mvn license:format on the root of the repo to automatically add missing headers.](https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L34) - `severity=info`
+3. [Every class, interface and enumeration should have JavaDoc describing its purpose and usage.](https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L66) - `severity=warning`
+4. [Every class, interface and enumeration must have an @author tag in its JavaDoc for every author that wrote a substantial part of the file.](https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L97) - `severity=error`
+5. Every constant, field and method with default, protected or public visibility should have JavaDoc (optional, but encouraged for private visibility as well). - `Not covered yet`
+6. The code must be formatted:
+    - java files must use spaces for formatting, rather than tabs - `severity=error`;
+    - xml files must use tabs for formatting, rather than spaces - `Not covered yet`
+    - json files must use tabs for formatting, rather than spaces - `Not covered yet`
+7. Generics must be used where applicable. - `Not covered yet`
+8. Code should not show any warnings. Warnings that cannot be circumvented should be suppressed by using the @SuppressWarnings annotation. - `Not covered yet`
+9. For dependency injection, OSGi Declarative Services should be used. - `severity=warning`
+10. Packages that contain classes that are not meant to be used by other bundles should have “internal” in their package name. - `Work in Progress`

```

Comment:
I remember that we have discussed to add the severity of the checks that are still work in progress, but not to mention them explicitly as work in progress.